src/content-script/listeners/player.ts

```diff
@@ -0,0 +1,110 @@
+import { debug } from '@root/lib/utils';
+import socket from '@root/lib/socket';
```

Comment:
Can we move this to `content-script/socket`? We'll only be accessing from there. I'll be removing from `popup` in another PR.